### ISSUE TITLE
Targeting ball improvements/fixes

### ISSUE THREAD

**Issue by sashan13** [PapaJoesSoup/BDArmory#405]
I am posting this here in addition to KSP forums so it doesn't get lost. 

So, I see you are now aiming FLIR ball basing on target coordinates in world-fixed coordinate system. This leads to significant jerking and lagging when the target is moving, and to it being unable to track airplanes and helis. I propose to make it track moving targets in a similar way as tracking radar geometry part does. Make it able to track aircraft. To do it, maybe use system of coordinates that is fixed to the parent vehicle but isn't rotating? This way it should be able to track targets that have low relative angular velocity with parent craft. 
Oh, also, ability to "track" sky would be good. If the target is outlined by sky and your craft is moving it is completely impossible to track it even when it's stationary.

**Comment by gomker** [comment]
Have you enabled "COM" track, that should track center of mass of any vessel locked.

**Comment by sashan13** [comment]
Of course, but implementation of that function makes it unable to reliably
track a vessel that moves faster than like 20m/s relative to terrain.

**Comment by endardamiian** [comment]
Hey i need some help cuz the targetingball is brocken
so the screen is just violett help
